__tests__/selectors-valid.css

```diff
@@ -5,3 +5,7 @@
 input[type="text"] {
 	line-height: 1.1;
 }
+
+.selector::hover {
```

Comment:
`:hover` is [pseudo-class](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes), can you change this to `::after` which is a [psuedo-element](https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements) which is what this rule change is testing.

Double colons are _only_ valid for _psuedo-elements_, single colons remain for _psuedo-classes_